Content.Server/Medical/HealingSystem.cs

```diff
@@ -188,6 +190,12 @@ private bool TryHeal(EntityUid uid, EntityUid user, EntityUid target, HealingCom
 
         var isNotSelf = user != target;
 
+        if (isNotSelf)
+        {
+            var msg = Loc.GetString("medical-item-popup-target", ("user", Identity.Entity(user, EntityManager)), ("item", uid));
+            _popupSystem.PopupEntity(msg, target, PopupType.Medium);
+        }
+
```

Comment:
Isn't this going to announce it for everyone in range? Makes the message seem weird.